.github/workflows/tests_primary.yml

```diff
@@ -78,6 +78,22 @@ jobs:
     - run: ./utils/upload_flakiness_dashboard.sh ./test-results/report.json
       if: always()
 
+  test_test_runner_windows_esm:
+    name: Test Runner Windows ESM
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 16
+    - run: npm i -g npm@8.3
+    - run: npm ci
+      env:
+        DEBUG: pw:install
+    - run: npm run build
+    - run: npx playwright install --with-deps
+    - run: npm run ttest -- --grep=esm
```

Comment:
Let's follow https://playwright.dev/docs/test-annotations#tag-tests:
`npm run ttest -- --grep=@esm`

Reply:
was doing it with `#` because we used it for our smoke tests: `#smoke`. Shall I change #smoke -> @esm in a follow-up too.

Reply:
Yeah, why not.